[bug] unresolved import `objc2_app_kit::NSImage`

### Describe the bug

<img width="805" alt="image" src="https://github.com/user-attachments/assets/de66b983-f8b2-429d-b72c-0afcfc8f275d">

rustc 1.81.0 (eeb90cda1 2024-09-04)
cargo 1.81.0 (2dbb1af80 2024-08-20)

### Reproduction

pnpm create tauri-app --rc baby

1. TypeScript
2. pnpm 
3. Svelte
4. TypeScript
5. pnpm install 
6. pnpm tauri dev

```
error[E0432]: unresolved import `objc2_app_kit::NSImage`
    --> /Users/developer/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tauri-2.0.0-rc.13/src/app.rs:2058:44
     |
2058 |         use objc2_app_kit::{NSApplication, NSImage};
     |                                            ^^^^^^^ no `NSImage` in the root

error[E0599]: no method named `setApplicationIconImage` found for struct `Retained<NSApplication>` in the current scope
    --> /Users/developer/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tauri-2.0.0-rc.13/src/app.rs:2067:24
     |
2067 |           unsafe { app.setApplicationIconImage(Some(&app_icon)) };
     |                        ^^^^^^^^^^^^^^^^^^^^^^^ method not found in `Retained<NSApplication>`

Some errors have detailed explanations: E0432, E0599.
For more information about an error, try `rustc --explain E0432`.
error: could not compile `tauri` (lib) due to 2 previous errors
```



### Expected behavior

_No response_

### Full `tauri info` output

```text
[✔] Environment
    - OS: Mac OS 14.5.0 x86_64 (X64)
    ✔ Xcode Command Line Tools: installed
    ✔ rustc: 1.81.0 (eeb90cda1 2024-09-04)
    ✔ cargo: 1.81.0 (2dbb1af80 2024-08-20)
    ✔ rustup: 1.27.1 (54dd3d00f 2024-04-24)
    ✔ Rust toolchain: stable-x86_64-apple-darwin (default)
    - node: 18.20.2
    - pnpm: 9.10.0
    - yarn: 1.22.22
    - npm: 10.5.0

[-] Packages
    - tauri 🦀: 2.0.0-rc.13
    - tauri-build 🦀: 2.0.0-rc.11
    - wry 🦀: 0.43.1
    - tao 🦀: 0.30.0
    - @tauri-apps/api : 2.0.0-rc.4
    - @tauri-apps/cli : 2.0.0-rc.15

[-] Plugins
    - tauri-plugin-shell 🦀: 2.0.0-rc.3
    - @tauri-apps/plugin-shell : 2.0.0-rc.1

[-] App
    - build-type: bundle
    - CSP: unset
    - frontendDist: ../dist
    - devUrl: http://localhost:1420/
    - framework: React
    - bundler: Vite
```


### Stack trace

_No response_

### Additional context

_No response_

--- a/crates/tauri/Cargo.toml
+++ b/crates/tauri/Cargo.toml
@@ -113,6 +113,7 @@ objc2-app-kit = { version = "0.2.2", features = [
   "NSResponder",
   "NSView",
   "NSWindow",
+  "NSImage",
 ] }
 window-vibrancy = "0.5"
 

